examples/inference/inference_KN.py

```
@@ -56,18 +59,20 @@
 likelihood = EMLikelihood(model,
                           data,
                           data_tmin=0.3,
-                          data_tmax=28.,
+                          data_tmax=10.,
                           trigger_time=trigger_time,
                           detection_limit = detection_limit,
                           fixed_params={"luminosity_distance": 43.583656, "redshift":0.009727})
 
 
 outdir = f"./outdir_KN/"
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)